### PR TITLE
Whisper Fix

### PR DESCRIPTION
**Pull request by chandralegend**

### CODE REVIEW

File: jaseci_ai_kit/jac_speech/jac_speech/stt/stt.py [hunk 1]
```diff
@@ -1,286 +1,130 @@
...
 @jaseci_action(act_group=["stt"], allow_remote=True)
 def transcribe(
-    language: str = "en",
-    array: List[float] = None,
     audio_file: str = None,
     url: str = None,
+    language: str = "en",
     timestamp: bool = False,
-) -> str:
+):
```
Comment (ypkang) [inline jaseci_ai_kit/jac_speech/jac_speech/stt/stt.py:31]: Lets add np array as an input option.

File: jaseci_ai_kit/jac_speech/jac_speech/stt/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
-transformers>=4.25.1
 librosa>=0.8.0,<=0.9.2
-soundfile>=0.11.0
-torch>=1.10.2,<2.0.0
\ No newline at end of file
+torch>=1.10.2
+openai-whisper
```
Comment (ypkang) [inline jaseci_ai_kit/jac_speech/jac_speech/stt/requirements.txt]: Make sure we have closed version requirements.

Comment (ypkang) [inline jaseci_ai_kit/jac_speech/jac_speech/stt/requirements.txt]: Make sure we have closed version requirements.

**Review by ypkang — CHANGES_REQUESTED** [review]

**Comment by chandralegend** [comment]
@ypkang STT works well in remote as well

**Review by ypkang — APPROVED** [review]

**Pull request merged**